staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -111,7 +111,7 @@ var (
 		&compbasemetrics.HistogramOpts{
 			Subsystem: APIServerComponent,
 			Name:      "request_slo_duration_seconds",
-			Help:      "Response latency distribution (not counting webhook duration and priority & fairness queue wait times) in seconds for each verb, group, version, resource, subresource, scope and component.",
+			Help:      "Response latency distribution (not counting webhook duration and priority & fairness queue wait times) in seconds for each verb, dry run value, group, version, resource, subresource, scope and component.",
```

Comment:
Since is a deprecated metric, I am not sure of the value gained by modifying it.